integration-test/tests/cms/footer.test.ts

```diff
@@ -1,14 +1,29 @@
 import {expect, test} from '@playwright/test';
 import { footerExpectedInformation } from '../../utils/datafactory/footer.data';
+import { validateSchema } from '../../utils/helpers/schema.validation';
+import { footerSchema } from '../../utils/datafactory/schemas/footer.schema';
+
 
 test('GET /api/cms/v1/footer returns correct footer data', async ({request}) => {
   const response = await request.get(`/api/cms/v1/footer`);
```

Comment:
@nimat1 please have a look here

Reply:
@nora-weisser @dricazenck - Do I need to install any dependencies inside the integration folder for my integration API's test  for schema validation, as I have added same format in my code-of conduct test and its giving me error regards to AVJ import  ?

Reply:
To update dependencies you have to run `npm install`.